src/components/article/ArticleList.jsx

```diff
@@ -25,7 +25,7 @@ function ArticleList({ topicSlug }) {
         <ul>
           {articles.map((article) => (
```

Comment:
Wrap these in a `<Link to={`/articles/${article.articls_id}`}` and you can use useParams rather than prop drilling your components.